soc/projects/science.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title:  Scientific Projects – Summer of Code
+---
+
+# {{ page.title }}
+
+{% include toc.html %}
+
+
+
+## Quantum Computation: Simualation of Noisy Circuits
+
+[Noisy Intermediate-Scale Quantum (NISQ) technology will be available in the near future.](https://arxiv.org/abs/1801.00862) However, it would be much more convenient if we could test our algorithm with noise and simulate our quantum algorithm on noisy circuits to explore their stability, efficiency. To assist the research of NISQ, enhance the quantum circuit simulator in Julia [Yao.jl](https://github.com/QuantumBFS/Yao.jl) with noisy circuit simulation would be quite useful. We are planning to implement the algorithm in a recent paper as our SoC project: [Efficient classical simulation of noisy quantum computation](https://arxiv.org/pdf/1810.03176.pdf).
+
+**Recommended Skills**: Background knowledge in quantum information and tensor networks and the ability of coding with Julia.
+**Expected Results**: Provide Yao.jl an extension package that defines different kinds of noise and provide Yao.jl with the ability to simulate certain kind of noisy circuits efficiently.
+**Mentors**: [Roger Luo](https://github.com/Roger-luo/)
+
+## Quantum Computation: Visualization of Quantum Circuits
+
+Although, there is already a pretty printing for quantum circuits in [Yao.jl](https://github.com/QuantumBFS/Yao.jl) as a tree, we are still lack of visualizing a quantum circuit defined as [block tree](https://quantumbfs.github.io/Yao.jl/latest/man/blocks/). And we have already have basic plotting utilities in Julia, like [Luxor.jl](https://github.com/JuliaGraphics/Luxor.jl). It would be great and more convenient to provide multiple theme for plotting a quantum circuit with Yao.jl to various formats.
+
+**Recommended Skills**: Basic knowledge in plotting in Julia with Luxor.jl.
```

Comment:
This might be a strong statement. I don't think you'd find a student with that background, and I'm not sure it's really necessary. Remember, GSoC is a first (major) Julia project for many students, let alone using something like Luxor.jl.

I'd say knowledge of programatic visualizations. Experience with Luxor.jl is preferred but can be learned along the way.

Reply:
I see, that's sound reasonable haha. I just personally found Luxor quite appealing.

Reply:
Yeah, I have changed it to a preferred skill. (You don't really need to "learn" Luxor, it is simple huh? ;-D )